module/ElasticSearch/src/ElasticSearch/VuFind/RecordDriver/ESSubject.php

```diff
@@ -272,4 +272,67 @@ public function getAllFields()
     {
         return $this->fields;
     }
+
```

Comment:
These properties were added to pass tag-cloud related information directly on the record driver. Otherwise the subjects bound to the renderer will not be an array of ESSubject objects which would result in problems when handled by the ESSubjectCollection view helper component

Reply:
Not happy :-(